boilerplate/app/services/reactotron/index.ts

```diff
@@ -1 +1,174 @@
-export * from "./reactotron"
```

Comment:
Given there's only one file, I'd recommend moving this and renaming it.

```
boilerplate/app/services/reactotron.ts
```

In general, I dislike `index.ts` (ever had 12 `index.ts` files open in the same VS Code instance...?). And if we're only doing a single file then a folder is redundant.

Reply:
I can get down with this. 

I think I may actually name this to `ReactotronConfig` since that's the convention in the docs: https://github.com/infinitered/reactotron/blob/master/docs/quick-start-react-native.md#configure-reactotron-with-your-project

Reply:
Created a new folder: `devtools` and put it into `app/devtools/ReactotronConfig` with this: https://github.com/infinitered/ignite/pull/2465/commits/1037d3ce290debf7e98cdbb203b443f51efe77c8